### ISSUE TITLE
p1203-standalone is not found

### ISSUE THREAD

**Issue by gdavila** [itu-p1203/itu-p1203#23]
`p1203-standalone` executable  is not found by the system after the installation through `pip3`. The last commit I tested and it worked was [6a76be0](https://github.com/itu-p1203/itu-p1203/tree/6a76be018eb979ccffa88c526c6baeeb86ac3d62) 

~~~
 ~# p1203-standalone
bash: /home/gabriel/.local/bin/p1203-standalone: No such file or directory
~~~

I'm on `Ubuntu 20.04.1 LTS` and `Python 3.8.2`.

**Comment by slhck** [comment]
Hm, interesting. The error message is weird. Why would it display the full path to something that does not exist?
How did you install it exactly? Maybe the executable is somewhere else?

**Comment by gdavila** [comment]
Apparently, that particular error message is because the  mlocate.db was not updated (but the file was previously created by the installation of the old commit version of [p1203](https://github.com/itu-p1203/itu-p1203/tree/6a76be018eb979ccffa88c526c6baeeb86ac3d62)). After a reboot, the message changes, but the executable is still missing:
~~~
 ~# p1203-standalone
p1203-standalone: command not found
~~~

Regarding the installation process, I'm just following the instructions:
~~~

pip3 install --user -r requirements.txt
pip3 install .
~~~

If I remove the p1203 with `pip3 uninstall itu_p1203` and then install and [older commit](https://github.com/itu-p1203/itu-p1203/tree/6a76be018eb979ccffa88c526c6baeeb86ac3d62), the `p1203-standalone` executable is rightly found.

**Comment by slhck** [comment]
Thanks for the additional detail! I think that the fact that a `pyproject.toml` file is present makes pip ignore the `setup.py` installation script. See also https://pip.pypa.io/en/stable/reference/pip/#pep-517-and-518-support.

Fixed in a0b8763202befa05983c1b9da853d0f56790d6f5
